yamlprivateh.go

```diff
@@ -83,15 +83,16 @@ func is_ascii(b []byte, i int) bool {
 
 // Check if the character at the start of the buffer can be printed unescaped.
 func is_printable(b []byte, i int) bool {
-	return ((b[i] == 0x0A) || // . == #x0A
+	//nolint:staticcheck // De Morgan is not needed here.
```

Comment:
Consider disabling QF1001 rule to avoid the DeMorgan's law suggestions.

I feel like it reported nothing interesting, and you add to disable them all with //nolint